.github/workflows/docker.yml

```diff
@@ -39,15 +45,6 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
-      - name: Restore Cached Docker Layers
-        id: restore-cache
-        uses: actions/cache/restore@v3
-        with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-cache-go-build-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-cache-go-build-
-
       - name: Prepare build args
         id: build_args
         run: |
```

Comment:
This comment is for L54. 
What's the value of `GITHUB_SHA` when the action is manually triggered. From doc it seems it's always the latest commit on a branch which can be different from the input commit hash.

Reply:
Ah good point. Perhaps a better option may be to simply skip saving the cache when we manually kick this off; I'll put a separate PR up for that later today

Reply:
Fixed in https://github.com/temporalio/docker-builds/pull/171